docs/testplan/SCP_copy-test-plan.md

```diff
@@ -0,0 +1,22 @@
+ - [Overview](#overview)
+     - [Scope](#scope)
+ - [Test Procedure](#test-procedure)
+
+## Overview
+The purpose of this test is to ensure that the DUT is able to fetch from an external SCP server without loss of file integrity. The file used by the test exceeds a gigabyte in order to ensure integrity even for large file sizes.
+
+### Scope
+This test was verified on a t0 topology, but should work on any other topology since the only interaction is between the host computer running the SCP server and the DUT.
+
+## Test Procedure
+
+1. The PTF container generates a 1 GB file with random bytes.
```

Comment:
Is 1 GB file necessary? We should add this test to kvm test script https://github.com/Azure/sonic-mgmt/blob/master/tests/kvmtest.sh. And to avoid too much resource hogging, we could use a smaller file size for this test

Reply:
Used 0.5 GB instead and also included in kvmtest.sh. Updated doc .